tests/ten/ten_cor_014/run.py

```diff
@@ -18,10 +18,26 @@ def execute(self):
         tx_receipt1 = network.transact(self, web3_usr, storage.contract.functions.store(1), account_usr, storage.GAS_LIMIT)
         tx_receipt2 = network.transact(self, web3_usr, storage.contract.functions.store(2), account_usr, storage.GAS_LIMIT)
 
+        # create another user and have them send funds to our test user
+        pk2 = self.get_ephemeral_pk()
+        web3_usr2, account_usr2 = network.connect(self, private_key=pk2, check_funds=False)
+        self.distribute_native(account_usr2, network.ETH_ALLOC_EPHEMERAL)  # Fund the second user
+    
+        # second user sends funds to our test user
+        transfer_amount = web3_usr2.to_wei(0.001, 'ether')
+        tx_receipt3 = network.transact(self, web3_usr2, {
```

Comment:
needs to be network.tx() ... the difference being network.transact works on target contract functions, whereas network.tx works generally on pre-built transaction dictionaries (not the best naming, but if you look at the method docs it is explained there)